Core/Src/stm32l0xx_it.c

```diff
@@ -85,7 +85,9 @@ void HardFault_Handler(void)
 {
   /* USER CODE BEGIN HardFault_IRQn 0 */
     __disable_irq();
+#ifdef SPI_DEBUG

```

Comment:
I don't think we should be adding our #defines to STM generated code. I fixed this problem by removing the #ifdef in iris_system.h